Cargo.lock

```diff
@@ -5293,13 +5293,13 @@ version = "0.42.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "5a3e1820f08b8513f676f7ab6c1f99ff312fb97b553d30ff4dd86f9f15728aa7"
 dependencies = [
- "windows_aarch64_gnullvm",
- "windows_aarch64_msvc",
- "windows_i686_gnu",
- "windows_i686_msvc",
- "windows_x86_64_gnu",
- "windows_x86_64_gnullvm",
- "windows_x86_64_msvc",
+ "windows_aarch64_gnullvm 0.42.2",
+ "windows_aarch64_msvc 0.42.2",
+ "windows_i686_gnu 0.42.2",
+ "windows_i686_msvc 0.42.2",
+ "windows_x86_64_gnu 0.42.2",
+ "windows_x86_64_gnullvm 0.42.2",
+ "windows_x86_64_msvc 0.42.2",
 ]
 
 [[package]]
```

Comment:
Should we / can we also bump this while we're at it? I'm not sure if that's a no-op change but it would be nice to avoid duplicating all these crates in our dependency tree.

Reply:
@Mark-Simulacrum Could you clarify if the objective is to minimize changes to unrelated dependencies and focus solely on upgrading linux-raw-sys? Would you be able to provide me with some guidance on how to proceed? Thank you. :heart:

Reply:
Were these changes generated by `cargo update -p is-terminal -p terminal_size` without any manual modifications. Should we trust the results generated by `cargo`? @Mark-Simulacrum